elements/pfe-band/test/pfe-band.spec.ts

```diff
@@ -4,6 +4,8 @@ import { hexToRgb, getColor } from '@patternfly/pfe-tools/test/hex-to-rgb.js';
 import { PfeBand } from '@patternfly/pfe-band';
 import { setViewport } from '@web/test-runner-commands';
 
+import '@patternfly/pfe-tools/test/stub-logger.js';
```

Comment:
Not sure what this is doing?  I don't see a console logger or a logger function inside the pfe-band component.

Reply:
What this does is stub out the Logger methods which call console methods. That way, we can still test them, but they don't pollute the test results output